core/src/com/unciv/models/ruleset/unique/Conditionals.kt

```diff
@@ -99,6 +99,48 @@ object Conditionals {
             return compare(statReserve, lowerLimit * gameSpeedModifier, upperLimit * gameSpeedModifier)
         }
 
+        fun getCountableAmount(countable: String): Float? {
+            if (countable.toIntOrNull() != null) return countable.toFloat()
```

Comment:
We don't actually need this line, since anything that can be converted to int can also be converted to float